virttest/utils_disk.py

```diff
@@ -790,6 +790,36 @@ def get_disk_size(session, os_type, did):
         return get_disk_size_windows(session, did)
 
 
+def get_drive_letters(session, did):
+    """
+    Get drive letters of specified disk in windows guest.
+
+    :param session: Windows VM session.
+    :type session: aexpect.ShellSession
+    :param did: The disk index.
```

Comment:
It looks like the paramater is the physical disk index? how to handle if it has multi partitions

Reply:
I will get all the letters.

Reply:
Hi @qingwangrh 
BTW, the name of the function is `get_drive_letters`, so it means get all the drive letters.
Thanks.